tests/bplan/test_bplan_api.py

```diff
@@ -248,3 +251,242 @@ def test_add_bplan_response(apiclient, organisation):
     assert response.data == {"id": pytest_regex("^[0-9]*$"), "embed_code": embed_code}
     assert len(mail.outbox) == 1
     assert mail.outbox[0].to == ["test@liqd.de"]
+
+
+@pytest.mark.django_db
+def test_add_bplan_diplan_response_no_embed(apiclient, districts, organisation):
+    url = reverse("bplan-list", kwargs={"organisation_pk": organisation.pk})
+    data = {
+        "name": "bplan-1",
+        "description": "desc",
+        "url": "https://bplan.net",
+        "start_date": "2013-01-01 18:00",
+        "end_date": "2021-01-01 18:00",
+        "identifier": "1-234",
+        "point": "[0,0]",
+    }
+    user = organisation.initiators.first()
+    apiclient.force_authenticate(user=user)
+    response = apiclient.post(url, data, format="json")
+    assert response.status_code == status.HTTP_201_CREATED
+    assert response.data == {"id": pytest_regex("^[0-9]*$")}
+
+
+@pytest.mark.django_db
+def test_bplan_api_adds_no_district_if_identifier_is_wrong(
+    apiclient, districts, organisation
+):
+    url = reverse("bplan-list", kwargs={"organisation_pk": organisation.pk})
+    data = {
+        "name": "bplan-1",
+        "description": "desc",
+        "url": "https://bplan.net",
+        "office_worker_email": "test@liqd.de",
+        "start_date": "2013-01-01 18:00",
+        "identifier": "A-AAAA",
+        "end_date": "2021-01-01 18:00",
+    }
+    user = organisation.initiators.first()
+    apiclient.force_authenticate(user=user)
+    response = apiclient.post(url, data, format="json")
+    assert response.status_code == status.HTTP_201_CREATED
+    bplan = Bplan.objects.first()
+    assert bplan.administrative_district is None
```

Comment:
so if a bplan can be saved without identifier, and it can be added later in an update, then in the custom `save()` we need to add the district flied in the `update_fields`